app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -126,3 +128,83 @@ define [
         else
           window.open(data[0].url)
       )
+
+    deleteRecording: (e) ->
+      e.preventDefault()
+      if !confirm I18n.t("Are you sure you want to delete this recording?")
+        $(e.currentTarget).parents('.inline-block').find('.al-trigger').focus()
+        return
```

Comment:
So if I understand this you are seeing if they haven't confirmed to delete the recording you pop a modal with that text then from there you put focus on what? I would like a comment here to tell us what is that focus otherwise we will be sad for a11y.

Reply:
Yes, this was a copy-paste from the delete method above. But it is not focusing on what it should. It may actually not be necessary. I'll double check the coffeescript test. Update the identifier if needed or remove the line.